backend/src/controllers/order_repository.ts

```diff
@@ -24,28 +24,28 @@ interface IOrderReposiotry {
 }
 
 export class MongoOrderRepository implements IOrderReposiotry {
-    async findAll(): Promise<GetOrdersResponse | null> {
+    async findAll(): Promise<GetOrdersResponse> {
```

Comment:
Remember to also modify the IOrderRepository

Reply:
I mean the return type.

Reply:
done!